subprojects/gradle/test-project/src/main/kotlin/com/avito/test/gradle/TestProjectGenerator.kt

```diff
@@ -65,6 +65,7 @@ class TestProjectGenerator(
                jcenter()
            }
         }
+        $buildGradleExtra
```

Comment:
It's strange, the first usage for plugins in the root project.

Reply:
I don't understand. Why is it strange for you?

Reply:
That we haven't faced with this obvious mistake.
Doesn't matter.